lib/src/benchmark_base.dart

```diff
@@ -1,64 +1,58 @@
 // Copyright 2011 Google Inc. All Rights Reserved.
 
-part of benchmark_harness;
+import 'score_emitter.dart' show PrintEmitter, ScoreEmitter;
 
 class BenchmarkBase {
   final String name;
   final ScoreEmitter emitter;
 
-  // Empty constructor.
   const BenchmarkBase(this.name, {this.emitter = const PrintEmitter()});
 
-  // The benchmark code.
-  // This function is not used, if both [warmup] and [exercise] are overwritten.
+  /// The benchmark code.
+  /// This function is not used, if both [warmup] and [exercise] are
```

Comment:
I would rewrite to:
The benchmark to be run.
The default implementations of [warmup] and [exercise] call this function.